packages/cli/tests/unit/agentInstall/javaBuildToolInstaller.spec.ts

```diff
@@ -16,20 +16,21 @@ describe('JavaBuildToolInstaller', () => {
   });
 
   describe('when IntelliJ is installed', () => {
-    const intellijHome = '/IntelliJ';
+    const intellijHome = (process.platform === 'win32' ? '\\' : '/') + 'IntelliJ';
     beforeEach(() => {
       sinon.stub(jbs, 'findIntelliJHome').returns(intellijHome);
     });
 
     it('appends JetBrains JDK to PATH', () => {
       addJetBrainsEnv();
-      expect(process.env.PATH).toMatch(/\/IntelliJ\/jbr\/bin/);
+      expect(process.env.PATH).toMatch(/[\/\\]IntelliJ[\/\\]jbr[\/\\]bin/);
```

Comment:
`toContain(path.join(intellijHome, 'jbr', 'bin'))` would also verify the right separator was used.